src/components/content/order/migrate/Migrate.tsx

```diff
@@ -30,8 +30,8 @@ export const Migrate = ({
     const [selectServiceHostingType, setSelectServiceHostingType] = useState<UserOrderableServiceVo.serviceHostingType>(
         currentSelectedService.serviceHostingType
     );
-    const [selectArea, setSelectArea] = useState<string>(''); // to be fixed.
-    const [selectRegion, setSelectRegion] = useState<string>(currentSelectedService.deployRequest.region);
+    const [selectArea, setSelectArea] = useState<string>(currentSelectedService.deployRequest.region.area); // to be fixed.
```

Comment:
can you please remove the comment here? This is exactly what was supposed to be fixed.